lib/v2/geocaching/blogs.js

```diff
@@ -0,0 +1,50 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    const response = await got('https://www.geocaching.com/blog/');
+    const data = response.data; // get html from the page
+
+    const $ = cheerio.load(data);
+    const list = $('div#primary main > article')
```

Comment:
Since this site is a WordPress based, you are suggested to use WordPress API instead of going through every pages.

Offical docs: https://developer.wordpress.org/rest-api/key-concepts/

All API endpoint: `/blog/wp-json/wp/v2`
Get posts: `/blog/wp-json/wp/v2/posts?per_page=100`